go/vt/sqlparser/testdata/select_cases.txt

```diff
@@ -9491,10 +9491,10 @@ OUTPUT
 select count(test.t1.b) from t1
 END
 INPUT
-select st_difference((convert(st_polygonfromwkb(linestring(point(1,1))) using gb18030)), st_geomcollfromwkb(point(1,1)));
+select st_difference((convert(st_polygonfromwkb(linestring(point(1,1), point(2,2))) using gb18030)), st_geomcollfromwkb(point(1,1)));
 END
 OUTPUT
-select st_difference(convert(st_polygonfromwkb(linestring(point(1, 1))) using gb18030), st_geomcollfromwkb(point(1, 1))) from dual
```

Comment:
Nevermind, it is already accepting single values. Just revert this change, so that we have a test for that too.

Reply:
Reverted